SeedAvatarStudio.sln

```diff
@@ -0,0 +1,20 @@
+﻿

```

Comment:
Typically we don't include "sln" files in git repo, since Unity can generate Visual Studio projects automatically.

You actually need to include a Unity-specific `.gitignore` file in this PR, to auto ignore files that we don't want to put into the repo. Please reuse the `.gitignore` file of the SeedCalc project.